.github/workflows/run_tests.yml

```diff
@@ -235,12 +235,9 @@ jobs:
       - name: Install go-swagger
         id: install-go-swagger
         run: |
-          sudo apt-get update
-          sudo apt-get install -y apt-transport-https gnupg curl
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/gpg.2F8CB673971B5C9E.key' | sudo apt-key add -
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/config.deb.txt?distro=debian&codename=any-version' | sudo tee /etc/apt/sources.list.d/go-swagger-go-swagger.list
-          sudo apt-get update
-          sudo apt-get install -y swagger
+          cd /usr/local/bin
+          wget -o swagger https://github.com/go-swagger/go-swagger/releases/download/v0.29.0/swagger_linux_amd64
```

Comment:
Might be worth saving this to `/usr/bin`. There's a step later in this flow that's failing on
```bash
if ! command -v swagger &> /dev/null
```

([source](https://github.com/CMSgov/easi-app/blob/314db4fbe1f112d494439094e7367d1ad20df957/scripts/generate_cedar_intake_client#L10))

Reply:
Good call - didn't catch that!